lib/sunrise/helper.rb

```diff
@@ -1,7 +1,36 @@
 module Sunrise
   module Helper
-    def html &block
-      # Implement me!
+    def html &code
+      instance_exec &code if block_given?
     end
+
+    private
+
+      def method_missing html_tag, *html_tag_attributes, &code
+        return html_tag_attributes.first.to_s if html_tag == :text
+
+        html = add_opening html_tag
+        html.insert -2, add_provided(html_tag_attributes) unless html_tag_attributes.empty?
+        html.concat yield if block_given?
+        html.concat add_closing html_tag
```

Comment:
Love the readability of `add_closing html_tag` 👌

Reply:
Me too. Det er dog ikke det hele der ser ud, som jeg gerne vil have det til. Det blev lidt sent i går, inden testene lyste grønt, så går nok lige over det igen. :)